.github/workflows/ci.yml

```diff
@@ -59,6 +59,10 @@ jobs:
           composer global require --no-interaction --no-progress symfony/flex:^1.11
           composer config extra.symfony.require ${{ matrix.symfony }}
 
+      - name: Remove EnqueueBundle
```

Comment:
This can be reversed when the enqueue packages support PHP 8. If you want I can add a TODO comment.